DP3TApp/Screens/Reports/NSReportsDetailNoReportsViewController.swift

```diff
@@ -67,7 +67,7 @@ class NSReportsDetailNoReportsViewController: NSTitleViewScrollViewController {
     }
 
     override var titleHeight: CGFloat {
-        return super.titleHeight * NSFontSize.fontSizeMultiplicator
+        return (super.titleHeight + NSPadding.large) * NSFontSize.fontSizeMultiplicator
```

Comment:
Why?

Reply:
Ain't nothin' but a heartache...
I also can't figure it out.